media/js/firefox/features-landing.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var supportsInsersectionObserver = typeof IntersectionObserver !== 'undefined';
+
+    function callback(changes, observer) {
```

Comment:
Nit - would be nice for this function to have a slightly less generic name. Maybe `observerCallback`?